1-js/02-first-steps/15-function-basics/article.md

```diff
@@ -218,16 +218,16 @@ function showMessage(from, text = anotherFunction()) {
 ```
 
 ```smart header="ارزیابی پارامترهای پیش‌فرض"
-در جاوااسکریپت، یک پارامتر پیش‌فرض هربار که تابع بدون پارامتر مریوطه صدا زده بشود، ارزیابی می‌شود.
+در جاوااسکریپت، یک پارامتر پیش‌فرض هربار که تابع بدون پارامتر مربوطه صدا زده شود، ارزیابی می‌شود.
 
-در مثال بالا، `anotherFunction()` هربار که `showMessage()` بدون پارامتر `text` صدا زده بشود، فراخوانی می‌شود.
+در مثال بالا، anotherFunction() هربار که showMessage() با پارامتر text صدا زده شود، فراخوانی نمی‌شود.
 
-از سوی دیگر، این به طور مستقل فراخوانی می‌شود وقتی `text` وجود نداشته باشد.
+از سوی دیگر، زمانی فراخونی می‌شود که text وجود نداشته باشد.
 ```
 
-### پارامترهای پیش‌فرض جایگزین
+### جایگزین پارامترهای پیش‌فرض 
```

Comment:
The text is translated from the [original text](https://github.com/javascript-tutorial/en.javascript.info/blob/master/1-js/02-first-steps/15-function-basics/article.md?plain=1#L228), you shouldn't change it on your own.